src/hospital/covid_examinations/models/examination.py

```diff
@@ -0,0 +1,19 @@
+"""Corona examination model."""
+from django.db import models
+
+from worker import Worker
+from patient import Patient
+
+
+class Examination(models.Model):
+    RESULTS = tuple({
+        "H": "Healthy",
+        "C": "Corona",
```

Comment:
I don't see the point in having shorten names such as "H" and "C" - we don't have storage problems and it would be easier to understand whats the actual value